internal/data/metric.go

```diff
@@ -74,9 +74,9 @@ func (md MetricData) MetricCount() int {
 	metricCount := 0
 	rms := md.ResourceMetrics()
 	for i := 0; i < rms.Len(); i++ {
-		ils := rms.Get(i).InstrumentationLibraryMetrics()
+		ils := rms.At(i).InstrumentationLibraryMetrics()
```

Comment:
Do we need ils.IsNil() check before calling isl.Len() or it is guaranteed not to be nil?

Reply:
Slice don't have an IsNil, they work mostly like slices in go, so len will return 0 for nil or empty slice.

Reply:
Ah, OK.